lib/web_ui/test/canvaskit/path_metrics_test.dart

```diff
@@ -3,6 +3,10 @@
 // found in the LICENSE file.
 
 // @dart = 2.6
+// This test failed on iOS Safari.
+// TODO: https://github.com/flutter/flutter/issues/60040
+@TestOn('chrome || firefox')
```

Comment:
We should still run it in desktop Safari. The issue seems to be limited to the iOS Simulator: https://bugs.webkit.org/show_bug.cgi?id=191064.

Reply:
Desktop safari is not running, there is a blocker on infra side.

Do you want to keep it for the local development?

Reply:
Yes, you are right. I tested it locally with Safari Desktop: `All tests passed!`.

I changed the test to only skip for iOS-Safari.